_security-plugin/configuration/security-admin.md

```diff
@@ -3,34 +3,67 @@ layout: default
+To initialize the `.opendistro_security` index, however, you must run `plugins/opensearch-security/tools/securityadmin.sh`. This script loads your initial configuration into the index using the configuration files in `plugins/opensearch-security/securityconfig`. After the `.opendistro_security` index is initialized, use OpenSearch Dashboards or the REST API to manage your users, roles, and permissions.
 
-After the `.opendistro_security` index is initialized, you can use OpenSearch Dashboards to manage your users, roles, and permissions.
+
+## A word of caution
+
+If you make changes to the configuration files in `plugins/opensearch-security/securityconfig`, OpenSearch does _not_ automatically apply these changes. Instead, you must run `securityadmin.sh` to load the updated files into the index.
+
+Running `securityadmin.sh` **overwrites** one or more portions of the `.opendistro_security` index. Run it with extreme care to avoid losing your existing resources. Consider the following example:
+
+1. You initialize the `.opendistro_security` index.
+1. You create ten users using the REST API.
+1. You decide to create a new [reserved user]({{site.url}}{{site.baseurl}}/security-plugin/access-control/api/#reserved-and-hidden-resources) using `internal_users.yml`.
+1. You run `securityadmin.sh` again to load the new reserved user into the index.
+1. You lose all ten users that you created using the REST API.
+
+To avoid this situation, back up your current configuration before making changes and re-running the script:
+
+```bash
+./securityadmin.sh -backup my-backup-directory \
+  -icl \
+  -nhnv \
+  -cacert ../../../config/root-ca.pem \
+  -cert ../../../config/kirk.pem \
+  -key ../../../config/kirk-key.pem
+```
+
+If you use the `-f` argument rather than `-cd`, you can load single YAML files into the index rather than the entire directory of YAML files. For example, if you create ten new roles, you can safely load `internal_users.yml` into the index without losing your roles; only the internal users get overwritten.
+
+```bash
+./securityadmin.sh -f ../securityconfig/internal_users.yml \
+  -t internalusers \
+  -icl \
+  -nhnv \
+  -cacert ../../../config/root-ca.pem \
+  -cert ../../../config/kirk.pem \
+  -key ../../../config/kirk-key.pem
+```
 
 
 ## Configure the admin certificate
 
-You can configure all certificates that should have admin privileges in `opensearch.yml` by specifying respective distinguished names (DNs). If you use the demo certificates, for example, you can use the `kirk` certificate:
+In order to use `securityadmin.sh`, you must add the distinguished names (DNs) of all admin certificates `opensearch.yml`. If you use the demo certificates, for example, `opensearch.yml` might contain the following lines for the `kirk` certificate:
```

Comment:
I think we're missing a "to" here? "...add the distinguished names (DNs) of all admin certificates to `opensearch.yml`."